src/drawing/area.rs

```diff
@@ -130,6 +130,14 @@ impl<DB: DrawingBackend, CT: CoordTranslate + Clone> Clone for DrawingArea<DB, C
     }
 }
 
+/// Sub error type specific to layout errors
+#[derive(Debug)]
+pub enum LayoutError {
+    /// Layout error coming from the Stretch library
+    StretchError(stretch::Error),
```

Comment:
Maybe a `#[cfg(feature = "stretch")]`. I think it's better to having this feature optional for now

Reply:
If I put `#[cfg(feature = "stretch")]` before that line, will it only be included in the enum if that feature is set?